### ISSUE TITLE
default decay value for BatchNormLayer

### ISSUE THREAD

**Issue by Jackqu** [zsdonghao/tensorlayer#104]
The current default decay value is 0.9 which may be too large, 0.999 maybe a better choice. What do you think?

**Comment by zsdonghao** [comment]
@Jackqu Hi, the default setting of TF is 0.999.

However, set decay to be a smaller value (decay=0.9 or decay=0.99) than default value (default is 0.999) if your dataset is small or your total training updates/steps are not that large.

reference: http://stackoverflow.com/questions/40879967/how-to-use-batch-normalization-correctly-in-tensorflow

**Comment by Jackqu** [comment]
Thank you for reply @zsdonghao

I had the same problem during inference time as described in http://stackoverflow.com/questions/40879967/how-to-use-batch-normalization-correctly-in-tensorflow.

 The inference result is much better when I set the is_training=True than when I set the is_training=False.
Initially I used the default decay value 0.9, so I guess it is the default decay value's problem. Then I tried 0.99 and 0.999, but both of these two values also have the same problem. The inference performance degrades significantly when is_training = False.  Has you had the same problem before?
